app/assets/javascripts/fae/navigation/_navigation.js

```diff
@@ -169,12 +169,13 @@ Fae.navigation = {
 
     if(FCH.exists('.js-content-header')) {
       var $header = $('.js-content-header');
-      var sidebar_top_offset = (parseInt( $header.outerHeight(), 10) + 20) + 'px';
-      $('#js-sidenav').css('padding-top',  sidebar_top_offset );
+      var sidebar_top_offset = (parseInt( $header.outerHeight(), 10) + 30) + 'px';
+      $('#js-sidenav').css('padding-top', sidebar_top_offset );
 
       $header.sticky({
         placeholder: true,
-        perpetual_placeholder: true
+        perpetual_placeholder: true,
+        ignore_placeholder_offsets: true
```

Comment:
This seems to add some unnecessary complexity. Do both have to be true? Couldn't `perpetual_placeholder` just be false? 

If `perpetual_placeholder` works as `false`, then you can remove the added `ignore_placeholder_offsets` option.